restful-ws/3.0/_index.md

```diff
@@ -1,24 +1,24 @@
 ---
-title: "Jakarta RESTful Web Services 3.0 (under development)"
-date: 2020-01-15
+title: "Jakarta RESTful Web Services 3.0 (milestone)"
+date: 2020-05-26
```

Comment:
FYI... This is the "go live" date.  So, once this PR is merged, if we past the date specified here, it will go live on the web site.  If, for some reason, you didn't want it to go live right away, you could push this date our further.  This allows you to merge at any time and determine when you want the page to go live.  Like I said, this is just an FYI comment -- using 05-26 is fine.

Reply:
Thanks - I did not know that.  I updated this to 2020-06-05 - an extra week or so to make corrections might be nice.